tensorflow/contrib/makefile/download_dependencies.sh

```diff
@@ -54,7 +54,7 @@ download_and_extract() {
   elif [[ "${url}" == *zip ]]; then
     tempdir=$(mktemp -d)
     tempdir2=$(mktemp -d)
-    wget -P ${tempdir} ${url}
+    wget --no-check-certificate -P ${tempdir} ${url}
```

Comment:
Why is this necessary?

Reply:
On my computer, it fails to download deps without this option:
downloading https://mirror.bazel.build/bitbucket.org/eigen/eigen/get/429aa5254200.tar.gz
downloading https://mirror.bazel.build/github.com/google/gemmlowp/archive/010bb3e71a26ca1d0884a167081d092b43563996.zip
--2017-11-03 14:25:06--  https://mirror.bazel.build/github.com/google/gemmlowp/archive/010bb3e71a26ca1d0884a167081d092b43563996.zip
Resolving mirror.bazel.build (mirror.bazel.build)... 130.211.22.235
Connecting to mirror.bazel.build (mirror.bazel.build)|130.211.22.235|:443... connected.
ERROR: cannot verify mirror.bazel.build's certificate, issued by ‘CN=Google Internet Authority G2,O=Google Inc,C=US’:
  Unable to locally verify the issuer's authority.
To connect to mirror.bazel.build insecurely, use `--no-check-certificate'.

Reply:
That's odd, but it seems unrelated to the CUDA functionality so should not be included in the PR. Is it possible that you're connecting through a proxy which is mangling the certificates?

Reply:
I just verified that I don't need this flag from home (Comcast).
I will remove this change and figure out what is going on with my working machine